src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -694,29 +694,23 @@ private void ReferToTempIfReferenceTypeReceiver(BoundLocal receiverTemp, ref Bou
                 temps.Add(intermediateRef.LocalSymbol);
                 extraRefInitialization = assignmentToTemp.Update(intermediateRef, assignmentToTemp.Right, assignmentToTemp.IsRef, assignmentToTemp.Type);
 
-                // If condition `(object)default(T) != null` is true at execution time,
-                // the T is a value type. And it is a reference type otherwise.
-                var isValueTypeCheck = _factory.ObjectNotEqual(
-                                            _factory.Convert(_factory.SpecialType(SpecialType.System_Object), _factory.Default(receiverType)),
-                                            _factory.Null(_factory.SpecialType(SpecialType.System_Object)));
-
                 // `receiverTemp` initialization is adjusted as follows:
                 // If we are dealing with a value type, use value of the intermediate ref.
                 // Otherwise, use an address of a temp where we store the underlying reference type instance.
                 assignmentToTemp =
                     assignmentToTemp.Update(
                         assignmentToTemp.Left,
-                        _factory.Conditional(
-                            isValueTypeCheck,
-                            intermediateRef,
-                            _factory.Sequence(new BoundExpression[] { _factory.AssignmentExpression(cache, intermediateRef) }, cache),
-                            receiverType,
-                            isRef: true),
+#pragma warning disable format
```

Comment:
nit: What's the formatting issue? Should we file a tracking issue?

Reply:
> What's the formatting issue? Should we file a tracking issue?

IDE wants the initializer on a separate line. I think it is by design, but I do not like the design. I have no intention to fight for its change though.